### PR TITLE
Version 1.1: Layout and Plugins

### PR DESCRIPTION
**Pull request by TimDiekmann**
Currently blocked on
- [x] #38 (PR: #62)
- [x] #54

### CODE REVIEW

**Comment by davidschlangen** [comment]
This page has all too many buttons and badges and colours and stuff. It's asking me to review something, but what?? 😄

**Comment by TimDiekmann** [comment]
At the top there is a tab "Changed files". There is the [changelog for v1.1](https://github.com/dsg-bielefeld/slurk/blob/5540f106abf4328d28aee956320a945d4169047f/CHANGELOG.md). :smile:

**Comment by TimDiekmann** [comment]
Is this changeset complete? Is something missing? Is something unclear? Formatting fine? (Next to README.md, the CHANGELOG.md will be read often)

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Version [1.1](https://github.com/dsg-bielefeld/slurk/tree/v1.1) (2019-26-03)
```
Comment (davidschlangen) [inline CHANGELOG.md]: Dates should be in ISO format (yyyy-mm-dd). The particular dates that occur here happen to be decodable unambiguously (on earth, with only 12 months), but others may not be... :-)

Comment (TimDiekmann) [reply CHANGELOG.md]: I'd say we should confine ourselves to Earth first. The internet is currently not sufficiently developed on other planets to use Slurk there. 

Furthermore we should generally release between the 13th and 31st, so we are on the safe side.

**Pull request merged**